pkg/template/generator/expressionvalue_test.go

```diff
@@ -16,8 +16,10 @@ func TestExpressionValueGenerator(t *testing.T) {
 		{"[\\d]{4}", "6841"},
 		{"[\\w]{20}", "UDd5PEJLyHZ7VpdMGC78"},
 		{"[\\a]{10}", "fUx4sLZWES"},
-		{"admin[0-9]{2}[A-Z]{2}", "admin05XX"},
```

Comment:
move the `generator` construction and seeding inside the loop so that changes don't affect all following tests